pallets/asset/src/benchmarking.rs

```diff
@@ -137,15 +138,15 @@ benchmarks! {
             registration_length: Some((60 * 24 * 60 * 60).into()),
         });
 
-        let caller = UserBuilder::<T>::default().build_with_did("caller", SEED);
+        let caller = UserBuilder::<T>::default().generate_did().build("caller");
         // Generate a ticker of length `t`.
         let ticker = Ticker::try_from(vec![b'A'; TICKER_LEN].as_slice()).unwrap(); }: _(caller.origin, ticker.clone()) verify {
         assert_eq!(Module::<T>::is_ticker_available(&ticker), false);
     }
 
     accept_ticker_transfer {
-        let owner = UserBuilder::<T>::default().build_with_did("owner", SEED);
-        let new_owner = UserBuilder::<T>::default().build_with_did("new_owner", SEED);
+        let owner = UserBuilder::<T>::default().generate_did().build("owner");
+        let new_owner = UserBuilder::<T>::default().generate_did().build("new_owner");
```

Comment:
I am not sure about that, because it is important to notice that `DID` is generated or not for the user.
BTW we could evaluate and do it in another PR.

Reply:
Let's call it `did(...)` or some such then. The main point is that there's substantial repetition that is noise; drowning out more important things.

Reply:
After an offline-chat, we are agree to use `make_did` instead of `generate_did`. 
att. @Centril